tests/run-local.sh

```diff
@@ -29,14 +33,63 @@ cleanExit () {
   "$SANDSTORM_DIR/sandstorm" stop
   sleep 1
   rm -rf "$SANDSTORM_DIR"
+  if [ -n "$XVFB_PID" ] ; then
+    # Send SIGINT to the selenium-server child of the backgrounded xvfb-run, so
+    # it will exit cleanly and the Xvfb process will also be cleaned up.
+    # We don't actually know that PID, so we find it with pgrep.
+    kill -SIGINT $(pgrep --parent $XVFB_PID node)
```

Comment:
This doesn't work on debian wheezy (specifically the -SIGINT), which is what our test box runs. Perhaps we should update...